lib/ansible/modules/network/cnos/cnos_rollback.py

```diff
@@ -200,67 +271,22 @@ def main():
             serverpassword=dict(required=False, no_log=True),),
         supports_check_mode=False)
 
-    username = module.params['username']
-    password = module.params['password']
-    enablePassword = module.params['enablePassword']
     outputfile = module.params['outputfile']
-    host = module.params['host']
-    deviceType = module.params['deviceType']
-    configType = module.params['configType']
     protocol = module.params['protocol'].lower()
-    rcserverip = module.params['serverip']
-    rcpath = module.params['rcpath']
-    serveruser = module.params['serverusername']
-    serverpwd = module.params['serverpassword']
-    output = ""
-    timeout = 90
-    tftptimeout = 450
-    if not HAS_PARAMIKO:
-        module.fail_json(msg='paramiko is required for this module')
-
-    # Create instance of SSHClient object
-    remote_conn_pre = paramiko.SSHClient()
-
-    # Automatically add untrusted hosts (make sure okay for security policy in your environment)
-    remote_conn_pre.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-
-    # initiate SSH connection with the switch
-    remote_conn_pre.connect(host, username=username, password=password)
-    time.sleep(2)
-
-    # Use invoke_shell to establish an 'interactive session'
-    remote_conn = remote_conn_pre.invoke_shell()
-    time.sleep(2)
-
-    # Enable and then send command
-    output = output + cnos.waitForDeviceResponse("\n", ">", 2, remote_conn)
-
-    output = output + cnos.enterEnableModeForDevice(enablePassword, 3, remote_conn)
-
-    # Make terminal length = 0
-    output = output + cnos.waitForDeviceResponse("terminal length 0\n", "#", 2, remote_conn)
-
-    # Invoke method for Config transfer from server
-    if(configType == 'running-config'):
-        if(protocol == "tftp" or protocol == "ftp"):
-            transfer_status = cnos.doRunningConfigRollback(protocol, tftptimeout, rcserverip, rcpath, serveruser, serverpwd, remote_conn)
-        elif(protocol == "sftp" or protocol == "scp"):
-            transfer_status = cnos.doSecureRunningConfigRollback(protocol, timeout, rcserverip, rcpath, serveruser, serverpwd, remote_conn)
-        else:
-            transfer_status = "Invalid Protocol option"
-    elif(configType == 'startup-config'):
-        if(protocol == "tftp" or protocol == "ftp"):
-            transfer_status = cnos.doStartUpConfigRollback(protocol, tftptimeout, rcserverip, rcpath, serveruser, serverpwd, remote_conn)
-        elif(protocol == "sftp" or protocol == "scp"):
-            transfer_status = cnos.doSecureStartUpConfigRollback(protocol, timeout, rcserverip, rcpath, serveruser, serverpwd, remote_conn)
-        else:
-            transfer_status = "Invalid Protocol option"
+    output = ''
+    if(protocol == "tftp" or protocol == "ftp" or
```

Comment:
If you like, something like this can be redone as `if protocol in ('tftp', 'ftp', 'sftp', 'scp'):`

Reply:
Sure, I will change that. I have requested for a mail from legal department. They are yet to respond. Will do this in meantime